packages/aws-cdk-lib/aws-logs/lib/log-retention.ts

```diff
@@ -148,7 +148,7 @@ class LogRetentionFunction extends Construct implements cdk.ITaggable {
     super(scope, id);
 
     const asset = new s3_assets.Asset(this, 'Code', {
-      path: path.join(__dirname, 'log-retention-provider'),
+      path: path.join(__dirname, '..', '..', 'custom-resource-handlers', 'dist', 'aws-logs', 'log-retention-handler'),
```

Comment:
Have you rebuilt `aws-cdk-lib` with `npx lerna run build --scope=aws-cdk-lib --skip-nx-cache`? The log retention handler was moved to `@aws-cdk/custom-resource-handlers/lib/aws-logs/log-retention-handler`, but when `aws-cdk-lib` is built the handler will be bundled and minified, and then we airlift all the bundled and minified handlers into the location pointed to by the updated code path you're referencing. I just cloned and built a fresh package without any problems to verify.

Reply:
Thank you for the explanation. I am still new to the codebase and I'm trying to run the tests directly from the test file. Since it is working fine on your side I will follow the instructions and hope to have success too 👍 Thank you for the fast reply 💯

Reply:
No problem. I'm happy to help!